test/universal/test-project/test.sh

```diff
@@ -116,7 +116,7 @@ check "fish" fish --version
 check "zsh" zsh --version
 
 # Check env variable
-check "RAILS_DEVELOPMENT_HOSTS is set correctly" echo $RAILS_DEVELOPMENT_HOSTS | grep ".githubpreview.dev,.app.github.dev"
+check "RAILS_DEVELOPMENT_HOSTS is set correctly" echo $RAILS_DEVELOPMENT_HOSTS | grep ".githubpreview.dev,.preview.app.github.dev,.app.github.dev"
```

Comment:
This check should fail 🤔 as https://github.com/devcontainers/images/pull/262/files isn't merged & released yet.
Looks like something is wrong with this test run 👀